app/pages/multisig/AccountTransactions/proposal-details/RegisterDataProposalDetails.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Account, Fraction } from '~/utils/types';
+import DisplayEstimatedFee from '~/components/DisplayEstimatedFee';
+import { AccountDetail, Details, PlainDetail } from './shared';
+import DisplayTransactionExpiryTime from '~/components/DisplayTransactionExpiryTime/DisplayTransactionExpiryTime';
+
+interface Props {
+    account?: Account;
+    estimatedFee?: Fraction;
+    expiryTime?: Date;
+    data?: string;
+}
+
+export default function AddBakerProposalDetails({
+    account,
+    estimatedFee,
+    expiryTime,
+    data,
+}: Props) {
+    return (
+        <Details>
+            <AccountDetail title="Account" value={account} first />
+            <DisplayEstimatedFee estimatedFee={estimatedFee} />
+            <PlainDetail title="Data" value={data} />
+            <DisplayTransactionExpiryTime expiryTime={expiryTime} />
```

Comment:
Would be nice if this showed "To be determined" when not specified yet. Either that, or the default value (I prefer the first option though).

Reply:
I just I prefer the first option as well.